services/apps/alcs/src/providers/typeorm/migrations/1691185932823-add_noi_submissions.ts

```diff
@@ -0,0 +1,179 @@
ype_description" text, "west_land_use_type" text, "west_land_use_type_description" text, "primary_contact_owner_uuid" text, "type_code" character varying NOT NULL, "created_by_uuid" uuid, CONSTRAINT "PK_373b13feac5362050544ee1dd62" PRIMARY KEY ("uuid")); COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."file_number" IS 'File Number of attached application'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."is_draft" IS 'Indicates whether submission is currently draft or not'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."applicant" IS 'The Applicants name on the application'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."local_government_uuid" IS 'UUID from ALCS System of the Local Government'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."purpose" IS 'The Applicants name on the application'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."parcels_agriculture_description" IS 'Quantify and describe in detail all agriculture that currently takes place on the parcel(s).'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."parcels_agriculture_improvement_description" IS 'Quantify and describe in detail all agricultural improvements made to the parcel(s).'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."parcels_non_agriculture_use_description" IS 'Quantify and describe all non-agricultural uses that currently take place on the parcel(s).'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."north_land_use_type" IS 'The land uses surrounding the parcel(s) under application on the North.'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."north_land_use_type_description" IS 'Description of the land uses surrounding the parcel(s) under application on the North.'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."east_land_use_type" IS 'The land uses surrounding the parcel(s) under application on the East.'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."east_land_use_type_description" IS 'Description of the land uses surrounding the parcel(s) under application on the East.'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."south_land_use_type" IS 'The land uses surrounding the parcel(s) under application on the South.'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."south_land_use_type_description" IS 'Description of the land uses surrounding the parcel(s) under application on the South.'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."west_land_use_type" IS 'The land uses surrounding the parcel(s) under application on the West.'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."west_land_use_type_description" IS 'Description of the land uses surrounding the parcel(s) under application on the West.'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."primary_contact_owner_uuid" IS 'Stores Uuid of Owner Selected as Primary Contact'; COMMENT ON COLUMN "alcs"."notice_of_intent_submission"."type_code" IS 'Notice of Intent Type Code'`,
```

Comment:
Might need to change the comment on purpose and applicant depending on response to previous MR feedback

Reply:
Added new migration